html2asketch/nodeToSketchLayers.js

```diff
@@ -144,10 +152,26 @@ export default async function nodeToSketchLayers(node) {
       await style.addImageFill(backroundImageToUrl(backgroundImage));
     }
 
-    style.addBorder({color: borderColor, thickness: parseInt(borderWidth, 10)});
-
     if (boxShadow !== DEFAULT_VALUES.boxShadow) {
-      style.addShadow(shadowStringToObject(boxShadow));
+      const shadowObj = shadowStringToObject(boxShadow);
+
+      if (boxShadow.indexOf('inset') !== -1) {
+        if (borderWidth.indexOf(' ') === -1) {
```

Comment:
not sure what's happening here - why are looking for space in border-width? Can you give me an example of box-shadow string that triggers this condition?

Reply:
It checks to use an independent one-side border.

If the widths of all 4 sides are the same,
borderWidth has only one value. => `1px`

But if css is this,
`` `
border: 1px solid # 000;
border-left: 3px solid # 000;
`` `

borderWidth has multiple values. => `1px 1px 1px 3px`

I tried to find a space in the string and check the one-side border.

Reply:
I'm sorry, I was wrong.
I will remove it.

I mistakenly thought that when I replaced the border with inner shadow, I did not need a spread offset.

Reply:
When one-side border and inner-shadow are applied together, the spread offset causes additional problems.

If I apply `parseInt` to the `borderWidth` of an independent border (ex` 3px 1px 1px 1px`)
it returns only the first value. (`3`).

If add this value to the spread of the inner-shadow, its result will be much different from html....

Reply:
Makes sense 👍